res/css/views/rooms/_BasicMessageComposer.scss

```diff
@@ -47,6 +47,7 @@ limitations under the License.
         &.mx_BasicMessageComposer_input_shouldShowPillAvatar {
             span.mx_UserPill, span.mx_RoomPill {
                 position: relative;
+                cursor: pointer;
```

Comment:
I think this wants to just target User Pills and not Room Pills as those don't have an onClick function